src/kbc_modules/cc_kbc/mod.rs

```diff
@@ -182,8 +182,8 @@ impl Kbc {
     }
 
     async fn request_kbs_resource(&mut self, resource_url: String) -> Result<Response> {
-        for attempt in 1..=KBS_GET_RESOURCE_MAX_ATTEMPT {
-            log::info!("CC-KBC: trying to get resource, attempt {attempt}");
+        for attempt in 0..=KBS_GET_RESOURCE_MAX_ATTEMPT {
```

Comment:
If we start counting from 0, the equal `=` on the right can be removed: 
```rust
for attempt in 0..KBS_GET_RESOURCE_MAX_ATTEMPT
```

Reply:
okay, you are right, let's close this one, I am still not familiar with the syntax of rust very well.